[2025-04-08T14:33:24.860+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-08T14:33:24.879+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T14:30:33.695401+00:00 [queued]>
[2025-04-08T14:33:24.886+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T14:30:33.695401+00:00 [queued]>
[2025-04-08T14:33:24.887+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-08T14:33:24.898+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-04-08 14:30:33.695401+00:00
[2025-04-08T14:33:24.907+0000] {standard_task_runner.py:63} INFO - Started process 378 to run task
[2025-04-08T14:33:24.910+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_streamlit', 'load_to_postgres', 'manual__2025-04-08T14:30:33.695401+00:00', '--job-id', '752', '--raw', '--subdir', 'DAGS_FOLDER/tmdb_pipeline.py', '--cfg-path', '/tmp/tmp2nd0t4e9']
[2025-04-08T14:33:24.913+0000] {standard_task_runner.py:91} INFO - Job 752: Subtask load_to_postgres
[2025-04-08T14:33:24.955+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T14:30:33.695401+00:00 [running]> on host 4b3aeb920c7e
[2025-04-08T14:33:25.041+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_streamlit' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T14:30:33.695401+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T14:30:33.695401+00:00'
[2025-04-08T14:33:25.042+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-08T14:33:25.066+0000] {tmdb_pipeline.py:1968} ERROR - No se encontró el archivo de datos procesados: None
[2025-04-08T14:33:25.089+0000] {tmdb_pipeline.py:1978} INFO - Leyendo 20 registros procesados para cargar en PostgreSQL
[2025-04-08T14:33:25.095+0000] {tmdb_pipeline.py:1551} INFO - Tablas creadas o verificadas correctamente en PostgreSQL
[2025-04-08T14:33:25.098+0000] {tmdb_pipeline.py:1994} INFO - Conexión establecida con PostgreSQL
[2025-04-08T14:33:25.126+0000] {tmdb_pipeline.py:2146} INFO - Total de películas en la base de datos: 62
[2025-04-08T14:33:25.127+0000] {tmdb_pipeline.py:2147} INFO - Insertadas 0 nuevas películas en esta ejecución
[2025-04-08T14:33:25.127+0000] {tmdb_pipeline.py:2153} INFO - Datos cargados correctamente en PostgreSQL
[2025-04-08T14:33:25.128+0000] {python.py:237} INFO - Done. Returned value was: 20
[2025-04-08T14:33:25.128+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-08T14:33:25.148+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=tmdb_pipeline_streamlit, task_id=load_to_postgres, execution_date=20250408T143033, start_date=20250408T143324, end_date=20250408T143325
[2025-04-08T14:33:25.173+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-08T14:33:25.200+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-08T14:33:25.202+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
